databend-jdbc/src/main/java/com/databend/jdbc/DatabendNodes.java

```diff
@@ -3,23 +3,28 @@
 import com.databend.client.ClientSettings;
 import com.databend.client.DatabendClientV1;
 import com.databend.client.DiscoveryNode;
+import lombok.Setter;
 import okhttp3.OkHttpClient;
 
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.security.InvalidParameterException;
 import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
-import java.util.stream.Collectors;
+import java.util.logging.Logger;
 
 public class DatabendNodes implements DatabendNodeRouter {
 
     private AtomicReference<List<URI>> query_nodes_uris;
     protected final AtomicInteger index;
     // keep track of latest discovery scheduled time
     protected final AtomicReference<Long> lastDiscoveryTime = new AtomicReference<>(0L);
+    private static final Logger logger = Logger.getLogger(DatabendNodes.class.getPackage().getName());
+    @Setter
+    private boolean debug = false;
```

Comment:
i guess if there is only one setter, define a `setDebug` is preferrable than using lomok